assets/src/edit-story/app/media/utils/getTypeFromMime.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Infer element type from mime type of its resource
+ *
+ * @param {string} mimeType Mime type.
+ * @return {string} Element type.
+ */
+const getTypeFromMime = (mimeType) => {
+  return mimeType.startsWith('image/') ? 'image' : 'video';
```

Comment:
The api can return other types then video and image. PDF, audio files, text files. This needs a better check.

Reply:
Do you suggest something specifically for this case considering our context? It was only a copy/paste migration. IMHO, it should return an error toast or something like that, but not sure if we have this type of feedback system covered. Thoughts?

Reply:
For now, I will use this approach https://github.com/google/web-stories-wp/blob/master/assets/src/edit-story/app/uploader/useUploader.js#L65-L67
assets/src/edit-story/app/uploader/useUploader.js:65-67

Reply:
Some example of mimetypes `image/png`, `video/mp4`. Can we use split. Something like this. 
```
const getTypeFromMime = (mimeType) => {
 const parts = mimeType.split('/');
 return parts.shift()
```